Makefile

```diff
@@ -1,13 +1,10 @@
 VERSION=$(shell git describe --tags --candidates=1 --dirty 2>/dev/null || echo "dev")
 FLAGS=-X main.Version=$(VERSION)
 
-all: darwin linux windows
+release-multi-arch: clean
```

Comment:
Thinking about this name, does it actually release them or is it just compiling binaries?

Reply:
just compiling. I'll change it to build

Reply:
Nice 👍  I think that would be clearer, it confused me a bit